Makefile.am

```diff
@@ -410,6 +410,30 @@ EXTRA_DIST += \
 	tools/autobahn \
 	data/eos-knowledge-0.pc.in
 
+# # # DOCUMENTATION # # #
+
+EXTRA_DIST += \
+	docs/api/domain.md \
+	docs/api/flatpak-bundle.md \
+	docs/api/hotdoc.json \
+	docs/api/index.md \
+	docs/api/sitemap.txt \
+	docs/framework/hotdoc.json \
+	docs/framework/index.md \
+	docs/framework/sitemap.txt \
+	docs/framework/tools.md \
+	$(NULL)
+
+HOTDOC_PROJECTS = docs/framework
```

Comment:
This is used inside `HOTDOC_MAKEFILE`.

I skipped generating `docs/api` so far, since we have almost nothing there and we're not going to publish it for the time being.